app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int):
```

Comment:
Best practice is to add type annotation for constructor as well, not only for arguments
-> None